packages/client/scripts/default-edge.d.ts

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
Would it make sense to use `never` here instead of `any`, so we'll have compile-time error (instead of runtime-only) if we try to use it without generating?

Reply:
I am reusing the what we currently ship as the default, happy to revisit this sometime soon.